### ISSUE TITLE
Can't return to login screen if user is autologin

### ISSUE THREAD

**Issue by fredcw** [linuxmint/wayland#13]
### Distribution

Mint 21.2 daily builds

### Package version

6.

### Frequency

Always

### Bug description

From wayland session, if user account is set to autologin, trying to logout, logs you out but immediately logs you back in to wayland session without showing the login screen making it impossible to change back to xorg session.

Opening login settings from the menu also doesn't work making it difficult to turn off autologin for user without using terminal.

Rebooting doesn't help because autologin.

Note: logout works OK on manjaro.

### Steps to reproduce

Set user to autologin using login window settings
Start wayland session.
Click logout in menu applet -> can't log out or change back to Xorg session.

### Expected behavior

return to login screen from wayland session even if user is autologin.

**Comment by fredcw** [comment]
Same on Mint 21.3 beta (cinnamon 6.0.2)

**Comment by haggen88** [comment]
> Same on Mint 21.3 beta (cinnamon 6.0.2)


@fredcw  perhaps you should also point out this bug in the current beta.

**Comment by mtwebster** [comment]
No, wayland support is experimental and not valid for the beta.

**Comment by fredcw** [comment]
@mtwebster Actually, now I think about, maybe this shouldn't be classed as a wayland bug but rather a bug with the display manager. The problem is not being able to log back into an Xorg session after trying wayland. While it's easy enough to fix on the command line if you know what you're doing, for some people, this could render their OS unusable.

As I said, it works fine on Manjaro so I suspect it's a problem with Mint's display manager config.

**Comment by mtwebster** [comment]
Are you using slick-greeter/lightdm on manjaro also?  I do remember testing auto-login.. not sure what happened here.

**Comment by mtwebster** [comment]
Does this still happen if you have two users?

**Comment by fredcw** [comment]
I'm at work atm so I'll have to check later. Yes, manjaro uses lightdm, not sure the greeter from memory.

**Comment by fredcw** [comment]
Yes, manjaro also uses slick greeter and using Mint with 2 users, logout works normally.

Edit: Correction, With 2 users, after a reboot, first attempt to logout works OK but all subsequent logouts, including after further reboots, don't work.

**Comment by timur-g** [comment]
> While it's easy enough to fix on the command line if you know what you're doing, for some people, this could render their OS unusable.

I am in the second group, left with black screen, user log out does not help.
It would be useful to write here how to fix.
And to take this as serious issue for users that just wanted to test.

**Comment by rcalixte** [comment]
> It would be useful to write here how to fix.

This was [posted on the Linux Mint Forums](https://forums.linuxmint.com/viewtopic.php?t=411761) and the blog. Can you verify that the instructions work for you?

**Comment by fredcw** [comment]
@timur-g A slightly easier way to do it is to open a terminal and type:

`sudo lightdm-settings`

On the "Users" tab, remove your user name from the Automatic login field and reboot.
